pkgs/development/tools/build-managers/gradle/fetch-deps.nix

```diff
@@ -123,10 +123,15 @@ let
 
             fileList = builtins.filter (x: lib.hasPrefix xmlBase x && x != url) (builtins.attrNames finalData);
             jarPomList = map parseArtifactUrl fileList;
-            sortedJarPomList =
-              lib.sort
-                (a: b: lib.splitVersion a.version < lib.splitVersion b.version)
-                jarPomList;
+
+            a-before-b = a: b:
```

Comment:
As the commit notes, this is inspired by [`pkgs/applications/science/math/mathematica/default.nix`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/science/math/mathematica/default.nix#L53-L58).